extraction/src/main/scala/org/broadinstitute/monster/encode/extraction/ExtractionPipelineBuilder.scala

```diff
@@ -191,25 +171,77 @@ class ExtractionPipelineBuilder(batchSize: Long, getClient: () => EncodeClient)
       .withName("Split by experiment type")
       .partition(isFunctionalCharacterizationReplicate)
 
-    extractLinkedEntities(
+    val experiments = extractLinkedEntities(
       sourceEntityType = EncodeEntity.Replicate,
       sourceField = "experiment",
       sourceEntities = expReplicate,
       targetEntityType = EncodeEntity.Experiment,
       targetField = "@id"
     )
-    extractLinkedEntities(
+    val fcExperiments = extractLinkedEntities(
       sourceEntityType = EncodeEntity.Replicate,
       sourceField = "experiment",
       sourceEntities = fcReplicate,
       targetEntityType = EncodeEntity.FunctionalCharacterizationExperiment,
       targetField = "@id"
     )
+
+    // Extracting files is too complicated to fit into the usual pattern.
```

Comment:
Ugh, I think I'm going to go back to pulling files based on the library. This seems to be pulling less relevant info (turns out "contributing files" is mainly references)

Reply:
... Or maybe our type-splitting is just broken